packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/default/index.tsx

```diff
@@ -0,0 +1,472 @@
+                )}
+              </DisplaySubTitle>
+            </BottomMultiRowContainer>
+          </BottomRow>
+          <BottomRow>
+            <DisplayIcon>
+              <Icon size='18px' color='grey800' name='pending' />
+            </DisplayIcon>
+            <BottomMultiRowContainer>
+              <DisplayTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.deposit.processing_time'
+                  defaultMessage='Processing Time'
+                />
+              </DisplayTitle>
+              <DisplaySubTitle>
+                {props.account.currency === 'GBP' && (
+                  <FormattedMessage
+                    id='modals.simplebuy.deposit.processing_time.info.gbp1'
+                    defaultMessage='Funds will be credited to your GBP Account as soon as we receive them. In the UK Faster Payments Network, this can take a couple of hours.'
+                  />
+                )}
+                {props.account.currency === 'EUR' && (
+                  <FormattedMessage
+                    id='modals.simplebuy.deposit.processing_time.info.eur1'
+                    defaultMessage='Funds will be credited to your EUR Account as soon as we receive them. SEPA transfers usually take around 1 business day to reach us.'
+                  />
+                )}
+                {props.account.currency === 'USD' && (
+                  <FormattedMessage
+                    id='modals.simplebuy.deposit.processing_time.info.usd1'
+                    defaultMessage='Funds will be credited to your USD Account as soon as we receive them. Funds are generally available within one business day.'
+                  />
+                )}
+              </DisplaySubTitle>
+            </BottomMultiRowContainer>
+          </BottomRow>
+
+          {props.account.currency === 'GBP' && (
+            <LegalWrapper inline>
+              <Text size='12px' weight={500} color='grey600'>
+                <FormattedMessage
+                  id='modals.simplebuy.transferdetails.depositagreement'
+                  defaultMessage='By depositing funds to this account, you agree to {ToS}, our banking partner.'
+                  values={{
+                    ToS: (
+                      <Link
+                        href='https://exchange.blockchain.com/legal'
+                        size='12px'
+                        weight={500}
+                        rel='noreferrer noopener'
+                        target='_blank'
+                      >
+                        <FormattedMessage
+                          id='modals.simplebuy.transferdetails.agree'
+                          defaultMessage='Terms and Conditions of Modular'
+                        />
+                      </Link>
+                    )
+                  }}
+                />
+              </Text>
+            </LegalWrapper>
+          )}
+        </BottomInfoContainer>
+      </Bottom>
+    </div>
+  )
+}
+
```

Comment:
tons of hardcoded EUR, GBP, USD strings.  can we replace with the fiat enums being used?